pages/dr8/bitmasks.rst

```diff
@@ -53,6 +53,8 @@ Bit Name          Description
 11  ``MEDIUM``    touches a pixel within the locus of a `radius-magnitude relation for Gaia DR2`_ stars `to G < 16`_
 12  ``GALAXY``    touches a pixel in an `SGA`_ large galaxy
 13  ``CLUSTER``   touches a pixel in a globular cluster
+14  ``SATUR_I``   touches a pixel that was saturated in at least one :math:`i`-band image
```

Comment:
Let's explicitly mention that these bits (will) only appear in DR10 and are zeroed out in earlier releases.

Reply:
Good suggestion. I noted that the i-band quantities would always be zero prior to DR10.